internal/services/redhatopenshift/redhat_openshift_cluster_resource.go

```diff
@@ -57,9 +57,10 @@ type ClusterProfile struct {
 }
 
 type NetworkProfile struct {
-	OutboundType string `tfschema:"outbound_type"`
-	PodCidr      string `tfschema:"pod_cidr"`
-	ServiceCidr  string `tfschema:"service_cidr"`
+	OutboundType            string `tfschema:"outbound_type"`
+	PodCidr                 string `tfschema:"pod_cidr"`
+	ServiceCidr             string `tfschema:"service_cidr"`
+	PreconfiguredNSGEnabled bool   `tfschema:"preconfigured_nsg_enabled"`
```

Comment:
(as below)

```suggestion
	PreconfiguredNetworkSecurityGroupEnabled bool   `tfschema:"preconfigured_network_security_group_enabled"`
```

Reply:
Apologies, I missed the fact that you made this change and I simply needed to commit.  Regardless, I fixed in commit becf28602323ab08a37def639367eb0975a74629